scripts/export_uml_json.py

```diff
@@ -0,0 +1,146 @@
+"""
+
+Convert a umls release to a json file of concepts.
+
+"""
+import sys
+import json
+import plac
+
+# preferred definitions (from S2)
+DEF_SOURCES_PREFERRED = {'NCI_BRIDG', 'NCI_NCI-GLOSS', 'NCI', 'GO', 'MSH', 'NCI_FDA'}
+
+
+@plac.annotations(
+        output_filename=('name of the output json file', 'positional', None, str),
+        umls_meta_directory=('path of the META directory', 'positional', None, str))
+def main(output_filename, umls_meta_directory):
+
+    concepts_filename = '{}/MRCONSO.RRF'.format(umls_meta_directory)
+    types_filename = '{}/MRSTY.RRF'.format(umls_meta_directory)
+    definitions_filename = '{}/MRDEF.RRF'.format(umls_meta_directory)
+    file_descriptors = '{}/MRFILES.RRF'.format(umls_meta_directory)  # to get column names
+
+    concepts_header = types_header = definitions_header = None
+    with open(file_descriptors) as fin:
```

Comment:
As this is code that reads data from a format no-one is familiar with, the format itself needs a bit more description.

Additionally, please separate this script into 4 functions:

`read_umls_file_descriptors`
`read_umls_concepts`
`read_umls_types`
`read_umls_definitions`

and move them into the library itself (e.g `scispacy/umls_utils.py`).

You could then write a small test and documentation for what each of these functions reads in the docstring of the function itself.

Reply:
done